### PR TITLE
provide error message when suppressionFile does not exist [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by MikeLing**

### CODE REVIEW

File: master/buildbot/test/regressions/test_steps_shell_WarningCountingShellCommand.py [hunk 1]
```diff
@@ -48,3 +49,10 @@ def testSuppressingLinelessWarningsPossible(self):
         # list of warnings
         expectedWarnings = 0
         self.assertEquals(len(warnings), expectedWarnings)
+
+    # To test if we can catch the exception when and return an error
+    # the suppressionFile does not exist or no right to access the file.
+    def testWithNotExistSuppressionFile(self):
+        # Use a filename which doesn't exist
+        w = WarningCountingShellCommand(
+            suppressionFile='impossiblefilename.txt')
```
Comment (MikeLing) [inline master/buildbot/test/regressions/test_steps_shell_WarningCountingShellCommand.py]: How could I make sure the values of  *stderr* and *rc* in here?

Comment (tardyp) [inline master/buildbot/test/regressions/test_steps_shell_WarningCountingShellCommand.py]: you can look in other tests for self.expectCommands there are plenty of example tests to simulate remote commands

**Comment by djmitche** [comment]
@MikeLing will you have a change to update this as well?

**Comment by MikeLing** [comment]
@djmitche , haven't yet. I need look into it more.

**Pull request closed without merging**